articles/fleetctl.md

```diff
@@ -102,9 +102,9 @@ Password:
 
 Once your local context is configured, you can use fleetctl normally.
 
-### Log in with SAML (SSO) authentication
+#### Users with sign sign-on (SSO) or email two-factor authentication (2FA)
```

Comment:
```suggestion
#### Log in with single sign-on (SSO) or email two-factor authentication (2FA)
```

Reply:
Do we want "Users" or "Log in" here? The change to "Users" was from @noahtalerman

Reply:
oh never mind, I see this is a subheading